website/docs/r/org.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_org"
+sidebar_current: "docs-vcd-resource-org"
+description: |-
+Provides a vCloud Director Organization resource. This can be used to create and delete an organization.
+---
+
+# vcd\_org
+
+Provides a vCloud Director Org resource. This can be used to create and delete an organization.
+
+Supported in provider *v2.0+*
+
+## Example Usage
+
+```
+resource "vcd_org" "my-org" {
+  name             = "my-org"
+  full_name        = "My organization"
+  description      = "The pride of my work"
+  is_enabled       = "true"
+  delete_force     = "true"
+  delete_recursive = "true"
```

Comment:
Please switch places of delete_force and delete_recursive parameters to better match the logical relationship between them:

```
delete_recursive = "true"
delete_force     = "true"  
```